src/ol/featureloader.js

```diff
@@ -87,8 +87,10 @@ ol.featureloader.tile = function(url, format) {
        * @this {ol.VectorTile}
        */
       function(features, dataProjection) {
-        this.setProjection(dataProjection);
-        this.setFeatures(features);
+        /** @type {ol.VectorTile} */
+        var me = this;
```

Comment:
What do you mean with grouping?

I just also try this but it's not working :-(
```
  /** 
   * @this {ol.VectorTile}
   * @param {Array.<ol.Feature>} features The loaded features.
   * @param {ol.proj.Projection} dataProjection Data projection.
   */
  var success = function(features, dataProjection) {
    this.setProjection(dataProjection);
    this.setFeatures(features);
  };  
  /** 
   * @this {ol.VectorTile}
   */
  var failure = function() {
    this.setState(ol.Tile.State.ERROR);
  };  
  return ol.featureloader.loadFeaturesXhr(url, format, success, failure);
```
The error message I have is:
```

ERR! compile /home/external/workspace/ngeo/node_modules/openlayers/src/ol/featureloader.js:89: ERROR - Property setProjection never defined on this
ERR! compile     this.setProjection(dataProjection);
ERR! compile          ^
ERR! compile 
ERR! compile 
ERR! compile /home/external/workspace/ngeo/node_modules/openlayers/src/ol/featureloader.js:90: ERROR - Property setFeatures never defined on this
ERR! compile     this.setFeatures(features);
ERR! compile          ^
ERR! compile 
ERR! compile 
ERR! compile 2
ERR! compile  error(s), 
ERR! compile 0
ERR! compile  warning(s), 
ERR! compile 96,4
ERR! compile %
ERR! compile  typed
ERR! compile 
ERR! compile 
ERR! Process exited with non-zero status, see log for more detail: 2 
```